README.md

```diff
@@ -6,6 +6,82 @@ multibot
 
 A programmatic multi-repository helper.
 
+Multibot allows you to invoke actions (`read`, etc.) on 1+ files in 1+
+repositories concurrently. It is meant to be a helper tool for multi-project
+updates, branches, pull requests, etc.
+
+## Install
+
+```sh
+$ npm install multibot
+```
+
+## Usage
+
+```
+A friendly multi-repository robot.
+
+Usage: multibot --action=<string> [options]
+
+Options:
+  --action       Actions to take                               [string] [required] [choices: "read"]
```

Comment:
I still think it makes sense to default to `read` since it's idempotent and probably the first thing you want to do to sanity check before actually creating commits.